WcaOnRails/config/routes.rb

```diff
@@ -26,6 +26,7 @@
   get 'admin/avatars' => 'admin/avatars#index'
   post 'admin/avatars' => 'admin/avatars#update_all'
 
+  get 'competitions/mycomps' => 'competitions#mycomps', as: :my_comps
```

Comment:
I don't think there's any value in putting "comps" in the url again, how about just `competitions/mine`?